src/redux/configureStore.js

```diff
@@ -1,12 +1,12 @@
-import { configureStore, combineReducers } from 'redux';
-import bookReducer from './books/books';
-import checkStatusReducer from './categories/categories';
+import { configureStore } from '@reduxjs/toolkit';
+import books from './books/books';
+import categories from './categories/categories';
 
-const reducer = combineReducers({
-  bookReducer,
-  checkStatusReducer,
+const store = configureStore({
+  reducer: {
+    books: books.reducer,
+    categories: categories.reducer,
+  },
```

Comment:
- [ ] [Optional] Kindly add redux-thunk to the project as a package and use it as a middleware. Eg:
```
  import { createStore, applyMiddleware } from 'redux';
    import thunk from 'redux-thunk';
    import rootReducer from './reducers/index';

    const store = createStore(
        rootReducer,
        applyMiddleware(thunk)
    );
```